OWM-CHAT Plugin Submission

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/Cerisk/OWM-Chat.git
```

Comment:
This file should not be added. The only one submitted should be `plugins/owm-chat`.